bip-musig2.mediawiki

```diff
@@ -335,8 +335,8 @@ Input:
 ** Let ''extra_in = empty_bytestring''
 * Let ''k<sub>i</sub> = int(hash<sub>MuSig/nonce</sub>(rand || bytes(1, len(aggpk)) || aggpk || m_prefixed || bytes(4, len(extra_in)) || extra_in || bytes(1, i - 1))) mod n'' for ''i = 1,2''
 * Fail if ''k<sub>1</sub> = 0'' or ''k<sub>2</sub> = 0''
-* Let ''R<sup>*</sup><sub>1</sub> = k<sub>1</sub>⋅G, R<sup>*</sup><sub>2</sub> = k<sub>2</sub>⋅G''
-* Let ''pubnonce = cbytes(R<sup>*</sup><sub>1</sub>) || cbytes(R<sup>*</sup><sub>2</sub>)''
+* Let ''R<sub>1,⁎</sub> = k<sub>1</sub>⋅G, R<sub>2,⁎</sub> = k<sub>2</sub>⋅G''
```

Comment:
Shouldn't it be `R<sub>⁎,1</sub>` to be consistent with nonc agg (where signer index is first)?

Reply:
Oh yes, the entire point here was to be consistent with nonce agg...

Reply:
I think it's correct. The signer index is in fact second in NonceAgg.

Reply:
??? no idea what I was looking at